Rakefile

```diff
@@ -1,5 +1,6 @@
 require 'bundler/gem_tasks'
 require 'rake/testtask'
+load './lib/tasks/check_about_yml.rake'
```

Comment:
Heh, I forgot about that... In #57, though, I wound up going the `require_relative` route.